content/en/real_user_monitoring/session_replay/mobile/_index.md

```diff
@@ -18,26 +18,97 @@ Mobile Session Replay expands visibility into your mobile applications by visual
+
+For further context, the following could be considered a wireframe:
+
+- A label displaying text is considered a "text" wireframe. It uses a rectangle (x, y, width, height) to define its position.
+- The application's background view is considered a "geometry" wireframe stretched across the entire app window. In most cases, it can be described as a basic rectangle of a single color.
+- Any other container view in the app with an opaque background is a "geometry" wireframe.
+- An image or icon displayed in the app is an "image" wireframe with a defined position and sometimes style (such as opacity or alpha).
+- A map view hierarchy consisting of dozens of views (for rendering shapes and annotations) can be merged into a single "image" wireframe.
+
+### Recording algorithm
+
+Wireframes are constructed during the process of traverseing the app's view-tree in a bottom-up (or generally "back-to-front") order and determining visible elements. In the below example, the iOS app screen is built with 78 native _views_, but it can be broken into 19 wireframes:
+
+{{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-1.png" alt="An example of how the iOS app screen contains 78 native views, but is made up of 19 wireframes." style="width:30%;">}}
+
+These wireframes can be sent to the player while **preserving their rendering order** (back-to-front) and **using absolute positioning** (in screen coordinates). There is no "tree structure", nor child-parent relationship between wireframes, therefore making the structure "flat".
+
+### Rendering algorithm
+
+To display a single replay frame, the player needs to iterate through all wireframes and render one another into the viewport using geometry information (x, y, width, height) from each wireframe. It must respect wireframes rendering order, so that succeeding ones overdraw existing portions of the viewport.
+
+The above screen can be reconstructed in 19 passes of the renderer:
+
+| Iteration | 1 | 2 | 3 |
+|-----------|---|---|---|
+| Viewport | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-1.png" alt="An example of a 'geometry' wireframe." style="width:100%;">}} | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-2.png" alt="An example of an 'image' wireframe." style="width:100%;">}} | {{< img src="real_user_monitoring/session_replay/mobile/how-it-works/how-it-works-iteration-3.png" alt="An example of a 'text' wireframe." style="width:100%;">}} |
+| Description | Draw "geometry" wireframe into viewport (rectangle + background color). | Draw "image" wireframe (rectangle + image with given UUID fetched from the Datadog backend). | Draw "text" wireframe (rectangle + "Twin Lake" text + font color, family and size). |
```

Comment:
```suggestion
```

Let me know what you think but I feel we can get rid of the in-table descriptions, that I feel are a bit too low level

Reply:
I would be ok with that if the visuals say enough!